caikit_nlp/modules/text_generation/peft_config.py

```diff
@@ -216,5 +276,15 @@ def get_peft_config(
         tuning_config=tuning_config,
         output_model_types=output_model_types,
     )
+    return peft_config
```

Comment:
this `create_hf_tuning_config` function should ideally be moved here. I dont know rationale for why its in peft_prompt_tuning.py . Let me know if we can move it here and then get rid of `cls` argument